pandas/io/tests/test_pytables.py

```diff
@@ -927,6 +927,42 @@ def test_encoding(self):
             result = store.select('df',Term('columns=A',encoding='ascii'))
             tm.assert_frame_equal(result,expected)
 
+    def test_latin_encoding(self):
+        # issue GH10366
+
+        if compat.PY2:
+            raise nose.SkipTest('HDF encoding not working in Python < 3')
+
+        values = [[b'E\xc9, 17', b'', b'a', b'b', b'c'],
```

Comment:
``unicode`` is not supported in PY2. So don't skip the test, rather assert that you can the error

``TypeError: [unicode] is not implemented as a table column``